infrastructure/quick-deploy/localhost/all/Makefile

```diff
@@ -0,0 +1,51 @@
+CURRENT_DIR:=$(shell pwd)
+GENERATED_DIR=$(CURRENT_DIR)/generated
+INGRESS_CERTIFICATES_DIR=$(GENERATED_DIR)/certificates/ingress
+PARAMETERS_FILE?=parameters.tfvars
```

Comment:
PARAMETERS_FILE?=terraform.tfvars

Reply:
not used

Reply:
I went back to `parameters.tfvars`. If I kept `terraform.tfvars`, I would not be able to use another file, even without using the Makefile.